src/node/nodetonode.h

```diff
@@ -49,6 +49,27 @@ namespace ccf
       channels = std::make_unique<ChannelManager>(network_pkey);
     }
 
+    template <class T>
+    void send(const NodeMsgType& msg_type, NodeId to, const T& data)
```

Comment:
I think it would be preferable to only have encrypted or integrity-protected messages flowing between nodes (that's what the status of a `n2n_channel` is: a symmetric key between two nodes has been computed via ECDH). Should we use `send_authenticated` and `recv_authenticated` for PBFT messages instead?

Reply:
Right now pbft performs it's own authentication of the received messages via the pre-verify() methods for each message. These pre-verify's for some types of messages check things that are needed by the pbft protocol.

Reply:
I agree with @jumaffre, let's have a discussion about this.